src/pass_manager.cpp

```diff
@@ -10,35 +10,34 @@
 #include <iostream>
 #include <sstream>
 #include <algorithm>
+#include <unordered_map>
 #include <utility>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_PASSES);
 
-void validate_pass(module& mod, const pass& p, tracer trace)
+void validate_pass(module& mod, const pass& p)
 {
     (void)mod;
     (void)p;
-    (void)trace;
 #ifndef NDEBUG
-    trace("Validate ...");
+    std::cout << "Validate..." << std::endl;
```

Comment:
This should print to the tracer otherwise it will always print out when using debug builds.